### DISCUSSION TITLE
Tagging of videos

### DISCUSSION THREAD

**Discussion by Rickrat** [Tyrrrz/YoutubeDownloader#203] [General]
I know that taglib-sharp isn't very good at video tagging, in fact, I have like 5 different versions of it on my machine and it does not work. I have found libmp4.net. It works really well and I modified the tagging service to use it.

[https://github.com/jasondavis303/libmp4.net](https://github.com/jasondavis303/libmp4.net)

**Comment by Tyrrrz** [comment]
Thanks, might have to take a look at that.
